compiler/test/compilable/test11980.d

```diff
@@ -1,2 +1,5 @@
-void start() {}
+void start()
+{
+	pragma(startaddress, start);
+}
 pragma(startaddress, start);
```

Comment:
Currently it doesn't check for it

Reply:
> surely having multiple start addresses should give an error? Or is this fine because its the same function for both?

Who cares, the pragma isn't implemented anyway. ;-)